libs/core/langchain_core/messages/utils.py

```diff
@@ -1294,13 +1302,26 @@ def convert_to_openai_messages(
 
     oai_messages: list = []
 
-    if is_single := isinstance(messages, (BaseMessage, dict, str, MessageV1Types)):
-        messages = [messages]
+    is_sequence = (
+        isinstance(messages, Sequence)
+        and not isinstance(messages, str)
+        and not (
+            isinstance(messages, tuple)
```

Comment:
A fix here: if messages is `tuple[str, list[dict]]`, it is a single `MessageLikeRepresentation`, not a sequence.